content/en/docs/tasks/run-application/run-replicated-stateful-application.md

```diff
@@ -15,7 +15,7 @@ weight: 30
 
 This page shows how to run a replicated stateful application using a
 [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) controller.
-The example is a MySQL single-master topology with multiple slaves running
+The example is a MySQL single-primary topology with multiple replicas running
 asynchronous replication.
```

Comment:
```suggestion
The example topology has a single primary host and multiple replicas, using asynchronous
row-based replication.
```
?

Reply:
I used "server" instead of "host" in some other places. I am kind of split on which is better, but think it should be consistent. I was thinking people say "MySQL server" a lot, but server makes me imagine an actual physical computer, and not just the server process first. We could "service" so it would be "single primary service and multiple replicas"

Reply:
I added a slightly modified change to the pull request.